plat/hisilicon/hikey960/hikey960_pm.c

```diff
@@ -295,6 +296,14 @@ static const plat_psci_ops_t hikey960_psci_ops = {
 int plat_setup_psci_ops(uintptr_t sec_entrypoint,
 			const plat_psci_ops_t **psci_ops)
 {
+	unsigned int id;
```

Comment:
Hmm, checker complained about "Using uninitialized value "id"."

You could init to PL011_UART6_BASE? But not blocking IMO.